operations/products.md

```diff
@@ -29,351 +32,361 @@ Returns all products offered together with the specified services. Note this ope
+| `ChargingMode` | [Product charging mode](products.md#product-charging-mode) | required | Charging mode of the product. |
+| `PostingMode` | [Product posting mode](products.md#product-posting-mode) | required | Charging mode of the product. |
+| `Options` | [Product options](products.md#product-options) | required | Options of the product. |
+| `Promotions` | [Promotions](services.md#promotions) | required | Promotions of the product. |
+| `Classifications` | [Product classifications](products.md#product-classifications) | required | Classifications of the product. |
+| `Price` | [Extended amount](_objects.md#amount) | required | Price representing price of the product. |
+| `ImageIds` | array of string | optional | Unique identifier of the product image. |
+| `Ordering` | integer | required | Order value for presentation purposes. |
+| `ExternalIdentifier` | string | optional, max length 255 characters | Identifier of the product from external system. |
+| `CreatedUtc` | string | required | Creation date and time of the product in UTC timezone in ISO 8601 format. |
+| `UpdatedUtc` | string | required | Last update date and time of the product in UTC timezone in ISO 8601 format. |
+| `ConsumptionMoment` | [Product consumption moment](products.md#product-consumption-moment) | required | Consumption moment of the product. |
+| ~~`IsDefault`~~ | ~~boolean~~ | ~~required~~ | **Deprecated!** |
+| ~~`Name`~~ | ~~string~~ | ~~optional~~ | ~~Name of the product. **Deprecated!** Please use Names~~ **Deprecated!** Use `Names` instead.|
+| ~~`ExternalName`~~ | ~~string~~ | ~~optional~~ | ~~Name of the product meant to be displayed to customer. **Deprecated!** Please use ExternalNames~~ **Deprecated!** Use `ExternalNames` instead.|
+| ~~`ShortName`~~ | ~~string~~ | ~~optional~~ | ~~Short name of the product. **Deprecated!** Please use ShortNames~~ **Deprecated!** Use `ShortNames` instead.|
+| ~~`Description`~~ | ~~string~~ | ~~optional~~ | ~~Description of the product. **Deprecated!** Please use Descriptions~~ **Deprecated!** Use `Descriptions` instead.|
+| ~~`Charging`~~ | ~~[Product charging mode](products.md#product-charging-mode)~~ | ~~required~~ | **Deprecated!** |
+| ~~`Posting`~~ | ~~[Product posting](products.md#product-posting)~~ | ~~required~~ | **Deprecated!** |
+| ~~`Pricing`~~ | ~~[Product pricing](products.md#product-pricing)~~ | ~~required~~ | **Deprecated!** Use `Price` instead.|
+
+#### Product charging mode
+
+* `Once`
+* `PerTimeUnit`
+* `PerPersonPerTimeUnit`
+* `PerPerson`
+
+#### Product posting
+
+* `Once`
+* `Daily`
+
+#### Product posting mode
+
+* `Once`
+* `PerTimeUnit`
+
+#### Product options
+Options of the product.
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `BillAsPackage` | boolean | required | Product should be displayed as part of a package. |
+
+#### Product classifications
+Classifications of the service.
```

Comment:
```suggestion
Classifications of the product.
```
Do we even need these descriptions when they don't provide more info than the title?

Reply:
Good catch, removed description from all the objects. 🚀